SimG4CMS/Calo/test/python/minbias_cfg.py

```diff
@@ -6,7 +6,7 @@
 process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi")
 process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
 process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
-process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load("Configuration.StandardSequences.MagneticField_38T_cff")
```

Comment:
I have also added these 3 lines in the cfg file apart from changing the B-field sequence:
process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
from Configuration.AlCa.GlobalTag import GlobalTag
process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run1_mc', '')


________________________________
From: David Lange [notifications@github.com]
Sent: 25 June 2015 18:19
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2 hcx14 bring in simualtion level of geometry changes to 7_X version (#9833)


In SimG4CMS/Calo/test/python/minbias_cfg.py<https://github.com/cms-sw/cmssw/pull/9833#discussion_r33273736>:

> @@ -6,7 +6,7 @@
>  process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi")
>  process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
>  process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
> -process.load("Configuration.StandardSequences.MagneticField_cff")
> +process.load("Configuration.StandardSequences.MagneticField_38T_cff")


MagneticField_cff should work with all current GTs in the alca matrix. If it didn’t, your GT was obsolete and should be updated instead of making this code change.
…
On Jun 25, 2015, at 5:26 PM, bsunanda <notifications@github.com> wrote: In SimG4CMS/Calo/test/python/minbias_cfg.py: > @@ -6,7 +6,7 @@ > process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi") > process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi") > process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi") > -process.load("Configuration.StandardSequences.MagneticField_cff") > +process.load("Configuration.StandardSequences.MagneticField_38T_cff") I understood that I needed some global tag for the B-field. It was giving error otherwise. While I had to change that I found 3.8T is a better option. I am not sure if this change was absolutely necessary for the run - but addition of data base and global tag was needed ________________________________ From: David Lange [notifications@github.com] Sent: 25 June 2015 17:21 To: cms-sw/cmssw Cc: Sunanda Banerjee Subject: Re: [cmssw] bsunanda:Run2 hcx14 bring in simualtion level of geometry changes to 7_X version (#9833<https://github.com/cms-sw/cmssw/pull/9833>) In SimG4CMS/Calo/test/python/minbias_cfg.py<https://github.com/cms-sw/cmssw/pull/9833#discussion_r33266857>: @@ -6,7 +6,7 @@ process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi") process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi") process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi") -process.load("Configuration.StandardSequences.MagneticField_cff") +process.load("Configuration.StandardSequences.MagneticField_38T_cff") @bsunanda<https://github.com/bsunanda><https://github.com/bsunanda> - why are you making this change? — Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/9833/files#r33266857>. — Reply to this email directly or view it on GitHub.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/9833/files#r33273736>.